linux/process.go

```diff
@@ -44,7 +46,15 @@ func (p *Process) State(ctx context.Context) (runtime.State, error) {
 		ID: p.id,
 	})
 	if err != nil {
-		return runtime.State{}, errdefs.FromGRPC(err)
+		if errors.Cause(err) != ttrpc.ErrClosed {
+			return runtime.State{}, errdefs.FromGRPC(err)
+		}
+
+		// TODO(stevvooe): We treat ttrpc.ErrClosed as the shim being closed,
```

Comment:
We don't set the process in a new group and with IO being owned by the shim the process is unable to survive without it.  You really don't need this as a TODO, just the explanation.

Reply:
I'll update it.